src/pages/profile-edit.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Profile Edit</title>
+ 
+  </head>
+  <body>
+    {{> header}}
+    <div class="flex justify-center bg-[#F5F5F5]">
+   <div class="m-10 flex flex-col gap-y-6 rounded  p-3 " dir="rtl">
+    <form class="flex flex-col gap-3  bg-[#FFFFFF] h-[528px] w-[986px] ">
+      <h2 class="w-[249px] h-[36px]  not-italic font-bold text-[20px] leading-[36px] m-5">ویرایش اطلاعات شخصی</h2>
+
+      <div class="grid grid-cols-3 gap-x-6 ">
+        <div class="flex flex-col gap-2 relative">
+          <label class="w-[88px] h-[17px]  not-italic font-light text-[12px] leading-[16.8px] text-[#7E8AAB] mr-5 mt-5">نام و نام خانوادگی</label>
+          <input type="text" class="w-[230px] h-[50px] bg-[#F7F9FC] text-right mr-5 pr-3" value="علیض">
+          <img  class="  absolute bottom-3 right-16 mr-2" src="/src/assets/images/pages/ui/crusore-icon.png" alt="crusore-image">
+        </div>
+        <div class="flex flex-col gap-2">
+          <label class="w-[88px] h-[17px]  not-italic font-light text-[12px] leading-[16 .8px] text-[#7E8AAB] mt-5">شماره موبایل</label>
+          <input type="number" class="w-[230px] h-[50px] bg-[#F7F9FC] text-right pr-3 " value="09152663045">
+        </div>
+        <div class="flex flex-col gap-2 relative ">
+          <label class=" w-[88px] h-[17px]  not-italic font-light text-[12px] leading-[16 .8px] text-[#7E8AAB] mt-5 ">آدرس ایمیل</label>
+          <input type="email" class="  w-[230px] h-[50px] bg-[#F7F9FC] text-right pr-3 " value="efwabifawafup@gmail.com ">
+          <img  class="  absolute bottom-3.5 left-7 ml-20 " src="/src/assets/images/pages/ui/crusore-icon.png" alt="crusore-image">
+        </div>
+      </div>
+  
+      <div class="grid grid-cols-2 gap-x-6">
+        <div class="flex flex-col gap-2 relative">
+          <label class="w-[88px] h-[17px]  not-italic font-light text-[12px] leading-[16 .8px] text-[#7E8AAB] mt-5 mr-5">استان</label>
```

Comment:
Use TailwindCSS classes.

```suggestion
          <label class="w-[88px] h-[17px]  not-italic font-light text-xs text-[#7E8AAB] mt-5 mr-5">استان</label>
```